content/2024/be-simple/index.md

```diff
@@ -98,11 +98,10 @@ Perhaps another way to explain it is the midwit theme:
 
 I try to stay on the left side of this curve as much as I can.
 
-That is, I'm aware that there are "smarter" ways to do things, but I don't want to dedicate time to learning about them.
-Knowing that getting over the middle-ground takes lots of effort keeps me on the left side. 
-The other end of the spectrum only provides diminishing returns anyway.
-I think I made it over that hump only a few times in my life.
-For the rest, knowing that there's an awkward middle-ground makes me keep things simple.
+I'm aware that there are "smarter" ways to do things, but I don't want to dedicate time to learning about them.
```

Comment:
Phrasing makes it seem like you don't want to learn / improve. But knowing you, you only want to spend your time on things that are important to you, not everything. So picking the things you go deep in. Might need some rephrasing still.

Reply:
Good point.

Reply:
updated